screens/Reader.js

```diff
@@ -635,13 +622,13 @@ class Reader extends React.Component {
                   onPress={() => {
                     const autoScrollSpeed = {
                       autoScroll: 0,
-                    scrollMultiplier,
+                      scrollMultiplier,
                     };
                     this.webView.postMessage(JSON.stringify(autoScrollSpeed));
                     this.setState({
                       paused: true,
                     });
-                    navigation.navigate('Settings');
+                    navigation.navigate("Settings");
```

Comment:
it should be refactored at all places and navigate tabs should be exported from a common file and be used wherever required.

Reply:
fixed